biz.aQute.bnd/bnd.bnd

```diff
@@ -33,6 +33,7 @@ Bundle-Description: This command line utility is the Swiss army knife of OSGi. I
 -buildpath: \
 	aQute.libg;version=latest,\
 	biz.aQute.bndlib;version=project,\
+	org.apache.felix.framework;version=latest;packages=*,\
```

Comment:
This seems very wrong. Why is this needed on the `-buildpath`? I see you use it in the test class, perhaps you meant to put it on the `-testpath`?